src/js/object-mona-parser/object-parser-common.js

```diff
@@ -26,17 +28,20 @@ function typeReferenceFromParsedDefinition(parsedTypeReference, genericsSection)
 function parseAttributeTypeReferenceSection() {
   return mona.sequence(function(s) {
     const typeReferenceSection = s(mona.trimLeft(mona.text(mona.or(mona.alphanum(), mona.string('_')))));
+    const protocolSection = s(mona.maybe(mona.between(mona.string('<'), mona.string('>'), mona.text(mona.or(mona.alphanum())))));
```

Comment:
We're going to have to find a better heuristic for choosing between generics and protocols at some point probably...